integration/jnifuse/fs/src/main/java/alluxio/jnifuse/AbstractFuseFileSystem.java

```diff
@@ -70,7 +75,7 @@ public AbstractFuseFileSystem(Path mountPoint) {
    */
   public void mount(boolean blocking, boolean debug, String[] fuseOpts) {
     if (!mMounted.compareAndSet(false, true)) {
-      throw new FuseException("Fuse File System already mounted!");
+      throw new FailedPreconditionRuntimeException("Fuse File System already mounted!");
```

Comment:
TODO(considering keeping JNIFUSE separate from Alluxio? and make it isolated, avoid changing the FuseException for now?